lib/plugins/plugin/install.js

```diff
@@ -1,201 +0,0 @@
-'use strict';
```

Comment:
@issea1015 sorry, one last thing (just realized).

We actually need to keep this internal plugin. Reason is that in case of _local fallback_. There can be scenario where older global version (which do not have yet `plugin install` command separated) will fallback to local version which has, having that removed will result with `command not found` error.

We will remove this completely with v3, where we would accept local fallback only from other v3+ releases

I would keep it as is, just add at front of this file comment: `TODO: Remove in v3`

Reply:
@medikoo No worries ~ I'm totally not in a rush haha. Let me do this!